pixleesdk/src/main/java/com/pixlee/pixleesdk/network/NetworkModule.java

```diff
@@ -111,10 +108,17 @@ public static Retrofit provideRetrofit(String url, OkHttpClient okHttpClient) {
 
     public static synchronized OkHttpClient provideOkHttpClient() {
         if (okHttpClient == null) {
+            Long timeoutConnect = 60L;
+            Long timeoutRead = 60L;
+            Long timeoutWrite = 180L;
+            if (PXLClient.Companion.getApiTimeoutConnect() != null) timeoutConnect = PXLClient.Companion.getApiTimeoutConnect();
+            if (PXLClient.Companion.getApiTimeoutRead() != null) timeoutRead = PXLClient.Companion.getApiTimeoutRead();
+            if (PXLClient.Companion.getApiTimeoutWrite() != null) timeoutWrite = PXLClient.Companion.getApiTimeoutWrite();
```

Comment:
If we set the default values on the companion object, we can clear this part up
```suggestion
            Long timeoutConnect = PXLClient.Companion.getApiTimeoutConnect();
            Long timeoutRead = PXLClient.Companion.getApiTimeoutRead();
            Long timeoutWrite = PXLClient.Companion.getApiTimeoutWrite();
```